docs/src/modules/components/TopLayoutBlog.js

```diff
@@ -231,7 +231,56 @@ function TopLayoutBlog(props) {
         disableAlternateLocale
         card={card}
         type="article"
-      />
+      >
+        <meta name="author" content={headers.authors.map((key) => authors[key].name).join(', ')} />
+        <meta property="article:published_time" content={headers.date} />
```

Comment:
Sounds like a battle between Facebook and Google with `og` standard vs `schema.org`. Since we do not know which service is using what, better to add them both

Reply:
👍